src/ccl/charts/line-graph.component.ts

```diff
@@ -11,7 +11,7 @@ import * as D3 from 'd3';
   selector: 'line-graph',
   encapsulation: ViewEncapsulation.None,
   template: `<ng-content></ng-content>`,
-  inputs: [ 'data', 'indicator' ],
+  inputs: [ 'data', 'indicator', 'addTrendline' ],
```

Comment:
This input might be clearer as `showTrendline`, `trendlineVisible` or even just `trendline`.